datahub-web-react/src/app/ingestV2/secret/SecretsList.tsx

```diff
@@ -341,8 +334,8 @@ export const SecretsList = ({ showCreateModal: isCreatingSecret, setShowCreateMo
                         />
                     </SearchContainer>
                 </StyledTabToolbar>
-                {tableData.length === 0 ? (
-                    <EmptyState />
+                {!loading && totalSecrets === 0 ? (
+                    <EmptySources sourceType="secrets" isEmptySearch={!!query} />
```

Comment:
yeah agreed with saketh that `isEmptySearch` and `!!query` don't match - `isEmptySearch` should mean there's no query? regardless i think we should make this a little clearer